#Upstream bugtracker entry: https://bugs.launchpad.net/screenlets/+bug/270648
#Comment: Partial patch from revision 102 of Universal-Applets branch (https://code.edge.launchpad.net/~screenlets-extras-team/universal-applets/universal-applets-extras).
#Copyright: Julien Lavergne (Original patch by Natan Yellin)
diff -ur screenlets-0.1.2.orig/src/share/screenlets/Places/PlacesScreenlet.py screenlets-0.1.2/src/share/screenlets/Places/PlacesScreenlet.py
--- screenlets-0.1.2.orig/src/share/screenlets/Places/PlacesScreenlet.py	2008-06-04 14:31:27.000000000 +0200
+++ screenlets-0.1.2/src/share/screenlets/Places/PlacesScreenlet.py	2008-09-17 01:15:40.000000000 +0200
@@ -110,7 +110,11 @@
 
 
 		
-		self.places =  [(os.environ['HOME'],'Home')]+ [(utils.get_desktop_dir(),'Desktop')] + utils.LoadBookmarks()
+		self.places =  [(os.environ['HOME'], 'Home'),
+						(utils.get_desktop_dir(),'Desktop')]
+		bookmarks = utils.LoadBookmarks()
+		if bookmarks:
+			self.places.extend(bookmarks)
 		#self.mounts = utils.readMountFile('/proc/mounts')
 		
 		if self.height != 20 + (len(self.places)*20) +20:
Seulement dans screenlets-0.1.2/src/share/screenlets/Places: PlacesScreenlet.py~
